src/map/script.c

```diff
@@ -12489,6 +12489,53 @@ static BUILDIN(hideonnpc)
 	npc->enable(str,4);
 	return true;
 }
+/*==========================================
+ *------------------------------------------*/
+static BUILDIN(cloakonnpc)
+{
+	struct npc_data *nd = npc->name2id(script_getstr(st, 2));
+	if (nd == NULL)
+		return false;
```

Comment:
I would show an error message if the NPC wasn't found, but that's up to you.

Reply:
returning false already prints an error, no need for another one.

Reply:
Well, return `false` just triggers `script_reportsrc()`.
And, in my opinion, a debug message like this isn't really helpful in that case:

> [Debug]: Source (NPC): Guard#DEV1 at prontera (147,169)